Wikipedia/Code/WMFHelpViewController.m

```diff
@@ -62,6 +63,10 @@ - (void)sendEmail {
         [vc setSubject:[WMFSettingsEmailSubject stringByAppendingString:[WikipediaAppUtils versionedUserAgent]]];
         [vc setToRecipients:@[WMFSettingsEmailAddress]];
         [vc setMessageBody:[NSString stringWithFormat:@"\n\n\n\nVersion: %@", [WikipediaAppUtils versionedUserAgent]] isHTML:NO];
+        NSData* data = [[DDLog wmf_currentLogFile] dataUsingEncoding:NSUTF8StringEncoding];
+        if (data) {
+            [vc addAttachmentData:data mimeType:@"text/plain" fileName:@"Log Data.txt"];
```

Comment:
maybe name "Item Counts.txt" so we're less tempted to use this a catch-all for log info?